sphinx/themes/basic/static/basic.css_t

```diff
@@ -537,6 +537,12 @@ dl.glossary dt {
     font-style: oblique;
 }
 
+.classifier:before {
+    font-style: normal;
+    margin: 0.5em;
+    content: ":";
+}
```

Comment:
In HTML4CSS1 writer, `<span class="classifier-delimiter">:</span>` is used for the delimiter. On the other hand, In HTML5-polyglot writer, it is represented by CSS hack.
Since 2.0, we adopt HTM5-polyglot style.